changelog_unreleased/markdown/pr-7220.md

```diff
@@ -0,0 +1,17 @@
+#### fix PR 7181 broke HTML blocks inside list items ([#7220](https://github.com/prettier/prettier/pull/7220) by [@sasurau4](https://github.com/sasurau4))
```

Comment:
This PR doesn't need a separate changelog entry. It can share one with #7181. Please remove this file and mention this PR in `pr-7181.md`:

```diff
-#### fix HTML formatting broken if the beginning tag starts after a list item ([#7181](https://github.com/prettier/prettier/pull/7181) by [@sasurau4](https://github.com/sasurau4))
+#### fix HTML formatting broken if the beginning tag starts after a list item ([#7181](https://github.com/prettier/prettier/pull/7181), [#7220](https://github.com/prettier/prettier/pull/7220) by [@sasurau4](https://github.com/sasurau4))
```

Reply:
Thanks for reviewing!